pkg/api/customization/kontainerdriver/store.go

```diff
@@ -0,0 +1,58 @@
+package kontainerdriver
+
+import (
+	"fmt"
+
+	errorsutil "github.com/pkg/errors"
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/tools/cache"
+)
+
+type Store struct {
+	types.Store
+	KontainerDriverLister v3.KontainerDriverLister
+	ClusterIndexer        cache.Indexer
+}
+
+func NewStore(management *config.ScaledContext, store types.Store) Store {
```

Comment:
since this store holds references to other objects, i think you want to return *Store instead of Store here.
Better, yet return the interface Store (from a different pacakge) and rename your struct store (lowercase) so that it isnt unnecessarily exported